metadata/_category/_template.md

```diff
@@ -0,0 +1,69 @@
+<!-- The filename should match the OpenSGID layer_name. For example, roads.md. The category will be inferred from the directory structure -->
```

Comment:
I like the idea of inferring, but I wonder if we need a category section for the cases when the data is not in OpenSGID.

Reply:
The "directory structure" refers to the structure within this repo, so each layer's .md file will live in a directory that corresponds to its category regardless of where the actual data are stored/exposed.